sql-jdbc/src/test/java/org/opensearch/jdbc/test/TLSServer.java

```diff
@@ -70,7 +70,7 @@ public static Server startSecureServer(
         ServerConnector httpsConnector = null;
 
         // setup ssl
-        SslContextFactory sslContextFactory = new SslContextFactory();
+        SslContextFactory.Server sslContextFactory = new SslContextFactory.Server();
```

Comment:
Is this constructor was deprecated 
https://opendev.org/opendev/gerrit/commit/4fa34cfbfcd04cce10ec8d595b669e31052de911
Thanks.